cli_tools/google-osconfig-agent/config/config.go

```diff
@@ -130,6 +130,18 @@ func createConfigFromMetadata(md metadataJSON) *config {
 	return c
 }
 
+func handleError(err error) error {
```

Comment:
nit: WDYT about renaming these `formatError()` or `fmtError` since "handle" is really generic and doesn't give much context?